src/core/dropdownmenu/dropdownmenu.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="dropdown is-hoverable">
+  <div class="dropdown-trigger">
+    <button class="button">
+      <span>{{ menuName }}</span>
```

Comment:
In some dropdowns you want to replace the dropdown menu name with the selected value, once you selected one.